scripts/editor/output-css-variables.js

```diff
@@ -117,6 +117,32 @@ export const outputCssVariablesGlobal = (globalManifest) => {
 	`);
 };
 
+/**
+ * Convert hex color into RGB values.
+ *
+ * @param {string} input - Input hex color (either 3 or 6 characters).
+ *
+ * @return {string}
+ */
+const hexToRgb = (input) => {
+	let r = 0, g = 0, b = 0;
+	const h = input.replace('#', '').trim();
+
+	if (h.length === 3) {
+		const [r1, g1, b1] = h;
+		r = `0x${r1}${r1}`;
+		g = `0x${g1}${g1}`;
+		b = `0x${b1}${b1}`;
+	} else if (h.length === 6) {
+		const [r1, r2, g1, g2, b1, b2] = h;
+		r = `0x${r1}${r2}`;
+		g = `0x${g1}${g2}`;
+		b = `0x${b1}${b2}`;
+	}
```

Comment:
What happens if the `h` length is something different than 3 or 6? I don't see an error here. Also, what does the `h` stands for?

Reply:
`h` = hex code

Tweaked the variable name now so it's more clear 😄